.github/workflows/kbs-rust.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         rust:
           - stable
           - beta
-          - nightly
+          - nightly-2024-02-18
```

Comment:
why fixing nightly to an (arbitrary?) date, would that give us any additional confidence?